tests/unit/test_select.py

```diff
@@ -239,63 +228,74 @@ def test_select_in_lit_13(faux_conn):
 
 
 @sqlalchemy_1_4_or_higher
-def test_select_in_lit(faux_conn):
-    [[isin]] = faux_conn.execute(
-        sqlalchemy.select([sqlalchemy.literal(1).in_([1, 2, 3])])
-    )
-    assert isin
-    assert _normalize_in_params(*faux_conn.test_data["execute"][-1]) == (
-        "SELECT %(p_0:INT64)s IN "
-        "UNNEST([ %(p_1:INT64)s, %(p_2:INT64)s, %(p_3:INT64)s ]) AS `anon_1`",
-        {"p_1": 1, "p_2": 2, "p_3": 3, "p_0": 1},
+def test_select_in_lit(faux_conn, last_query):
+    faux_conn.execute(sqlalchemy.select([sqlalchemy.literal(1).in_([1, 2, 3])]))
+    last_query(
+        "SELECT %(param_1:INT64)s IN UNNEST(%(param_2:INT64)s) AS `anon_1`",
```

Comment:
Should the type of `param_2` be `ARRAY<INT64>`?

Reply:
No, because the BQ DB API does special handling of arrays.

It sees that we have a scalar type of INT64 and that we have a sequence, and then creates a `ArrayQueryParameter`.

It happens that since I added struct support, passing `ARRAY<INT64>` would _probably_ work (because I have to handle structs of arrays).  But just usng `INT64` works too.